### DISCUSSION TITLE
Is the a way to use multi-word sentences in shortcuts?

### DISCUSSION THREAD

**Discussion by khomutovspace** [datenstrom/community#461] [Ask a question] [answered]
Is there any way to use sentence as one argument for extensions shortcuts. Now it uses every single word as separate argument. But what if it is needed to be one big sentence (like usual text) to appear somewhere using the shortcut?

**Comment by schulle4u** [comment, marked as answer]
Just enclosure your arguments with quotes. 

```
[example "This is an example argument" defaultstyle]
```

HTH
Steffen

**Comment by BsNoSi** [comment]
Strictly speaking, all parameters should be set in quotation marks. The accepted omission of unambiguous parameters is basically a typical slacker of programmers.
